[2025-06-09T07:50:37.908+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-09T07:50:37.937+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:50:34.908092+00:00 [queued]>
[2025-06-09T07:50:37.944+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:50:34.908092+00:00 [queued]>
[2025-06-09T07:50:37.944+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-09T07:50:37.967+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_from_netsuite> on 2025-06-09 07:50:34.908092+00:00
[2025-06-09T07:50:37.974+0000] {standard_task_runner.py:63} INFO - Started process 42 to run task
[2025-06-09T07:50:37.977+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'netsuite_saved_search_4532_memory', 'extract_from_netsuite', 'manual__2025-06-09T07:50:34.908092+00:00', '--job-id', '368', '--raw', '--subdir', 'DAGS_FOLDER/netsuite_4532_dag.py', '--cfg-path', '/tmp/tmpxm7ni8_z']
[2025-06-09T07:50:37.981+0000] {standard_task_runner.py:91} INFO - Job 368: Subtask extract_from_netsuite
[2025-06-09T07:50:38.036+0000] {task_command.py:426} INFO - Running <TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:50:34.908092+00:00 [running]> on host 46960f15b31b
[2025-06-09T07:50:38.122+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='netsuite_saved_search_4532_memory' AIRFLOW_CTX_TASK_ID='extract_from_netsuite' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T07:50:34.908092+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T07:50:34.908092+00:00'
[2025-06-09T07:50:38.123+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-09T07:50:40.806+0000] {logging_mixin.py:188} INFO - üîç Login status code: 200
[2025-06-09T07:50:40.807+0000] {logging_mixin.py:188} INFO - üîç Content-Type: text/html;charset=utf-8
[2025-06-09T07:50:40.808+0000] {logging_mixin.py:188} INFO - üîç Response sample: <!DOCTYPE html>
<html data-scroll-behavior="natural" lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	<script type="importmap">
		{"imports": {"@uif-js/builder":"/assets/@uif-js/builder/7.0.0/min/in
[2025-06-09T07:50:40.808+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-09T07:50:40.809+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/netsuite_4532_dag.py", line 61, in extract_from_netsuite
    raise Exception("‚ùå ƒêƒÉng nh·∫≠p NetSuite th·∫•t b·∫°i ho·∫∑c tr·∫£ v·ªÅ HTML")
Exception: ‚ùå ƒêƒÉng nh·∫≠p NetSuite th·∫•t b·∫°i ho·∫∑c tr·∫£ v·ªÅ HTML
[2025-06-09T07:50:40.827+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=netsuite_saved_search_4532_memory, task_id=extract_from_netsuite, run_id=manual__2025-06-09T07:50:34.908092+00:00, execution_date=20250609T075034, start_date=20250609T075037, end_date=20250609T075040
[2025-06-09T07:50:40.842+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 368 for task extract_from_netsuite (‚ùå ƒêƒÉng nh·∫≠p NetSuite th·∫•t b·∫°i ho·∫∑c tr·∫£ v·ªÅ HTML; 42)
[2025-06-09T07:50:40.854+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-09T07:50:40.883+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-09T07:50:40.885+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
